cmake/CMakeLists.txt

```diff
@@ -340,7 +340,7 @@ if (onnxruntime_USE_ROCM)
     if (ROCM_VERSION_DEV VERSION_LESS "6.2")
       message(FATAL_ERROR "CMAKE_HIP_ARCHITECTURES is not set when ROCm version < 6.2")
     else()
-      set(CMAKE_HIP_ARCHITECTURES "gfx908;gfx90a;gfx1030;gfx1100;gfx1101;gfx942;gfx950;gfx1200;gfx1201")
+      set(CMAKE_HIP_ARCHITECTURES "gfx908;gfx90a;gfx1030;gfx1100;gfx1101;gfx942;gfx950;gfx1200;gfx1201;gfx1150;gfx1151;gfx1152")
```

Comment:
I don't remember precisely. Is `gfx12-generic`/etc. supported as of ROCM 6.4?

Reply:
Should be. Ive taken a SHA-1 later than ROcm 6.4 as well